app/src/main/AndroidManifest.xml

```diff
@@ -23,6 +23,9 @@
     xmlns:tools="http://schemas.android.com/tools"
     package="com.amaze.filemanager">
 
+    <permission android:name="com.amaze.filemanager.permission.CONTROL_FTP_SERVER"
+        android:protectionLevel="signature" />
```

Comment:
protectionLevel="signature" is a bit much isn't it?

Reply:
Seems `dangerous` can still prevent unintended intents sent to `FtpReceiver`.